config/muxcable.py

```diff
@@ -188,3 +188,69 @@ def mode(state, port, json_output):
                 click.echo(tabulate(data, headers=headers))
 
         sys.exit(CONFIG_SUCCESSFUL)
+
+@muxcable.group(cls=clicommon.AbbreviationGroup)
+def prbs():
+    """Enable prbs on a port"""
+    pass
+
+@prbs.command()
+@click.argument('port', required=True, default=None, type=click.INT)
+@click.argument('target', required=True, default=None, type=click.INT)
+@click.argument('mode_value', required=True, default=None, type=click.INT)
+@click.argument('lane_map', required=True, default=None, type=click.INT)
+def enable(port, target, mode_value, lane_map):
+
+    import sonic_y_cable.y_cable
+    res = sonic_y_cable.y_cable.enable_prbs_mode(port, target, mode_value, lane_map)
+    if res != True:
+        click.echo("PRBS config unsuccesful")
+        sys.exit(CONFIG_FAIL)
+    click.echo("PRBS config sucessful")
+    sys.exit(CONFIG_SUCCESSFUL)
+
+@prbs.command()
+@click.argument('port', required=True, default=None, type=click.INT)
+@click.argument('target', required=True, default=None, type=click.INT)
+def disable(port, target):
+
```

Comment:
Suggest to add a docstring or `help=` parameter here to add a message in help dialogs (e.g., Suggest `"""Disable PRBS mode on a port"""`).

Reply:
fixed